### PR TITLE
Source Klaviyo: adds stream Templates

### PR DESCRIPTION
**Pull request by geekwhocodes**
## What
Adds new stream `Templates` to the source Klaviyo mentioned in this issue #17392
It uses existing HTTP Base Stream to include `templates` stream, read more here - [https://developers.klaviyo.com/en/v1-2/reference/get-templates](https://developers.klaviyo.com/en/v1-2/reference/get-templates) 

## Recommended reading order
2. `y.python`

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [x] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

### CODE REVIEW

**Comment by geekwhocodes** [comment]
@YowanR This is ready to review.

File: airbyte-integrations/connectors/source-klaviyo/source_klaviyo/schemas/email_templates.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "type": "object",
+    "properties": {
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-klaviyo/source_klaviyo/schemas/email_templates.json:3]: please add additionalProperties = true as other schemas file.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by geekwhocodes** [comment]
@marcosmarxm I made the required changes.

**Review by marcosmarxm — APPROVED** [review]
Thanks @geekwhocodes

**Comment by marcosmarxm** [comment]
@lazebnyi can you do the final review doing changes to a GA connector?

**Review by lazebnyi — APPROVED** [review]
Nice!

**Pull request merged**